.circleci/config.yml

```diff
@@ -1,34 +1,22 @@
 version: 2
 
-# NOTE: We run these in CircleCI because it has better artifacts support.
-# TODO: Move to GitHub Actions when its artifacts support improves.
+# NOTE: We run these in CircleCI because it has better artifacts support
+# and it is possible to publish image diffs as HTML like pytest-mpl in the future.
 jobs:
 
   image-tests-mpl302:
     docker:
-      - image: astropy/image-tests-py37-mpl302:1.10
+      - image: circleci/python:3.8
     steps:
       - checkout
       - run:
           name: Install dependencies
-          command: pip install -e .[test] pytest-mpl
-      - run:
-          name: Force scipy upgrade
-          command: pip install scipy -U
-      - run:
-          name: Run tests
-          command: pytest -P visualization --remote-data=astropy --open-files --mpl --mpl-results-path=$PWD/results -W ignore:np.asscalar
-      - store_artifacts:
-          path: results
-
-  image-tests-mpl310:
-    docker:
-      - image: astropy/image-tests-py37-mpl311:1.10
-    steps:
-      - checkout
-      - run:
-          name: Install dependencies
-          command: pip install -e .[test] pytest-mpl
+          command: |
+            pip install setuptools pip wheel --upgrade
+            pip install numpy==1.18.5
+            pip install scipy
+            pip install matplotlib==3.0.2
+            pip install -e .[test] pytest-mpl
```

Comment:
What about doing this all from tox, too? Or that's unnecessary as we would never reuse those envs locally/elsewhere?

Reply:
Yes I think defining those environments in tox makes sense

Reply:
Are you sure this wasn't a deliberate attempt to not use tox because we found a problem running pytest directly or something? If that does not ring a bell, I can define a tox env for this, but I want to make sure we're not undoing something else. I mean, why didn't @astrofrog define it in tox when he did the initial port? `mpldev` uses tox below, so it is not like we didn't port this file.

Reply:
we didn't yet use tox at that point if I recall correctly